src/librustc_mir/borrow_check/mod.rs

```diff
@@ -2093,7 +2159,7 @@ impl<'cx, 'gcx, 'tcx> MirBorrowckCtxt<'cx, 'gcx, 'tcx> {
         while let Some(i) = elems_incoming.next() {
             let borrowed = &data[i.borrow_index()];
 
-            if self.places_conflict(&borrowed.borrowed_place, place, access) {
+            if self.places_conflict(&borrowed.borrowed_place, &place, access) {
```

Comment:
Why this diff?

Reply:
I have no idea what that came from. My bad.